runtime/compiler/control/JITServerHelpers.cpp

```diff
@@ -123,11 +123,12 @@ JITServerHelpers::printJITServerMsgStats(J9JITConfig *jitConfig)
    PORT_ACCESS_FROM_JITCONFIG(jitConfig);
    j9tty_printf(PORTLIB, "JITServer Message Type Statistics:\n");
    j9tty_printf(PORTLIB, "Type# #called TypeName\n");
-   const ::google::protobuf::EnumDescriptor *descriptor = JITServer::MessageType_descriptor();
+   // const ::google::protobuf::EnumDescriptor *descriptor = JITServer::MessageType_descriptor();
    for (int i = 0; i < JITServer::MessageType_ARRAYSIZE; ++i)
       {
       if (JITServerHelpers::serverMsgTypeCount[i] > 0)
-         j9tty_printf(PORTLIB, "#%04d %7u %s\n", i, JITServerHelpers::serverMsgTypeCount[i], descriptor->FindValueByNumber(i)->name().c_str());
+         // j9tty_printf(PORTLIB, "#%04d %7u %s\n", i, JITServerHelpers::serverMsgTypeCount[i], descriptor->FindValueByNumber(i)->name().c_str());
```

Comment:
I left it there because we would still want to re-add the ability to print out message type name, instead of just its number. I can't think of a good way to do it other than creating an array of strings though. Should I add it now or later?

Reply:
Yes we need to add the array of strings. You can add it now because there is no additional risk.
I was also thinking about compacting the enum of message types,basically don't assign numbers to the entries.